cli/cmd/inject.go

```diff
@@ -188,7 +182,7 @@ func (rt resourceTransformerInject) transform(bytes []byte) ([]byte, []inject.Re
 		return b, reports, err
 	}
 	if b, _ := report.Injectable(); !b {
-		if e, err := shouldInjectThrowError(report); e {
+		if err := throwInjectError(report); err.Error() != "" {
```

Comment:
We don't use `err != nil` here because `throwInjectError` initializes a non-nil error (an empty string) for the sake of accumulation of error messages.